package.json

```diff
@@ -58,7 +58,7 @@
     "react-native-skeleton-content-nonexpo": "^1.0.13",
     "react-native-svg": "^13.6.0",
     "react-native-url-polyfill": "^1.3.0",
-    "react-native-vision-camera": "^2.15.2",
+    "react-native-vision-camera": "github:samvanhoeyicapps/react-native-vision-camera#v2.15.2-patch",
```

Comment:
Wanted to fork it so it would be very obvious that this was a temporary solution and a further upgrade would require more work. But a patch-package is also a possibility, worth a discussion